packages/app/components/Header/Header.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+import Image from "next/image";
+import Logo from "../../public/assets/images/logo.svg";
+import Stackly from "../../public/assets/images/stackly.svg";
+import Blocks from "../../public/assets/images/blocks.svg";
+
+import Link from "next/link";
+import MobileMenu from "./MobileMenu";
+
+export default function Header() {
+  return (
+    <header className="h-[72px] sticky top-0 z-50 flex flex-col w-full bg-surface-25 border-b border-solid border-b-surface-75">
```

Comment:
I don't think we want the nav `sticky`, at least by default.  Also z-index is to high, let start with `z-10`.

```suggestion
    <header className="h-[72px] top-0 z-10 flex flex-col w-full bg-surface-25 border-b border-solid border-b-surface-75">
```